src/main/java/com/foodapp/restaurant/RestaurantService.java

```diff
@@ -0,0 +1,42 @@
+package com.foodapp.restaurant;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class RestaurantService {
+
+    private final RestaurantRepository repository;
+
+    public RestaurantService(RestaurantRepository repository) {
+        this.repository = repository;
+    }
+
+    public List<Restaurant> getAllRestaurants() {
+        return repository.findAll();
+    }
+
+    public Optional<Restaurant> getRestaurantById(String id) {
+        return repository.findById(id);
+    }
```

Comment:
Is there a concern for data sanitation before the id is passed to the service? such as converting string and parsing as integer?

Reply:
The id is stored as a String in the database, so there's no need to parse it.